app/App/Main/Account/Requests/TransactionRequest/AdjustFee/index.js

```diff
@@ -1,12 +1,32 @@
-import React from 'react'
+import React, { Component, useState } from 'react'
 import Restore from 'react-restore'
+import BigNumber from 'bignumber.js'
 
-import svg from '../../../../../../../resources/svg'
 import link from '../../../../../../../resources/link'
-import { gweiToWei, gweiToWeiHex } from '../../../../../../../resources/utils'
+import { gweiToWei, gweiToWeiHex, intToHex } from '../../../../../../../resources/utils'
 import { usesBaseFee } from '../../../../../../../resources/domain/transaction'
 
-import BigNumber from 'bignumber.js'
+function toDisplayFromWei (bn) {
+  return bn.shiftedBy(-9).toFixed(9).toString().replace(/0+$/, '').replace(/\.+$/, '')
```

Comment:
are there BN methods we can use to do all this trimming instead of doing it manually with multiple regexes?

Reply:
Probably, didn't look at this function yet